store/config.go

```diff
@@ -26,26 +26,27 @@ import (
 // Config holds parameters used by the application which can be overridden
 // by setting environment variables.
 type Config struct {
-	LogLevel                 LogLevel        `env:"LOG_LEVEL" envDefault:"info"`
-	RootDir                  string          `env:"ROOT" envDefault:"~/.chainlink"`
-	Port                     string          `env:"CHAINLINK_PORT" envDefault:"6688"`
-	EthereumURL              string          `env:"ETH_URL" envDefault:"ws://localhost:8546"`
+	AllowOrigins             string          `env:"ALLOW_ORIGINS" envDefault:"http://localhost:3000,http://localhost:6688"`
 	ChainID                  uint64          `env:"ETH_CHAIN_ID" envDefault:"0"`
 	ClientNodeURL            string          `env:"CLIENT_NODE_URL" envDefault:"http://localhost:6688"`
-	MinIncomingConfirmations uint64          `env:"MIN_INCOMING_CONFIRMATIONS" envDefault:"0"`
-	MinOutgoingConfirmations uint64          `env:"MIN_OUTGOING_CONFIRMATIONS" envDefault:"12"`
+	DatabaseTimeout          Duration        `env:"DATABASE_TIMEOUT" envDefault:"500ms"`
+	Dev                      bool            `env:"CHAINLINK_DEV" envDefault:"false"`
 	EthGasBumpThreshold      uint64          `env:"ETH_GAS_BUMP_THRESHOLD" envDefault:"12"`
 	EthGasBumpWei            big.Int         `env:"ETH_GAS_BUMP_WEI" envDefault:"5000000000"`
 	EthGasPriceDefault       big.Int         `env:"ETH_GAS_PRICE_DEFAULT" envDefault:"20000000000"`
+	EthereumURL              string          `env:"ETH_URL" envDefault:"ws://localhost:8546"`
 	LinkContractAddress      string          `env:"LINK_CONTRACT_ADDRESS" envDefault:"0x514910771AF9Ca656af840dff83E8264EcF986CA"`
+	LogLevel                 LogLevel        `env:"LOG_LEVEL" envDefault:"info"`
+	MinIncomingConfirmations uint64          `env:"MIN_INCOMING_CONFIRMATIONS" envDefault:"0"`
+	MinOutgoingConfirmations uint64          `env:"MIN_OUTGOING_CONFIRMATIONS" envDefault:"12"`
 	MinimumContractPayment   big.Int         `env:"MINIMUM_CONTRACT_PAYMENT" envDefault:"1000000000000000000"`
+	MinimumRequestExpiration uint64          `env:"MINIMUM_REQUEST_EXPIRATION" envDefault:"300"`
```

Comment:
Maybe I'm thinking about this the wrong way, but shouldn't the expiration config be a maximum? 

For example:
As a node operator,
I want to set a maximum request expiration time that I will accept,
So that I know how long my penalty deposits will be locked for.

Reply:
We probably want both, but if it's too low then an operator could lose reputation if all of the requests they're supposed to fulfill close out. (e.g. they have an expiration of 1 second) Definitely worth adding a max field also, but could probably be it's own story.